protocols/grpc/agent.proto

```diff
@@ -110,6 +111,87 @@ message ListProcessesResponse {
 	bytes process_list = 1;
 }
 
+message StatsContainerRequest {
+    string container_id = 1;
+}
+
+message CpuUsage {
```

Comment:
Do we have a reference about why these stats are collected? IOW, are they sufficient to support the CRI metrics as well as the kata cli events?